src/C-interface/dense/bml_allocate_dense.c

```diff
@@ -5,6 +5,30 @@
 #include "bml_allocate_dense.h"
 #include "bml_types_dense.h"
 
+
+#ifdef BML_USE_MAGMA
+static magma_queue_t queue;
```

Comment:
Do I understand this correctly that we are going to use one global queue now as opposed to one per matrix?